pkgs/by-name/ld/ldproxy/package.nix

```
@@ -0,0 +1,36 @@
+{ lib, rustPlatform, fetchFromGitHub, pkg-config, udev, stdenv }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ldproxy";
+  version = "0.31.4";
+
+  # The source of this tools actually comes from the set of tools located in the `esp-rs:embuild` repository
+  # cf: https://github.com/esp-rs/embuild
+  src = fetchFromGitHub {
+    owner = "esp-rs";
+    repo = "embuild";
+    rev = "v${version}";
+    sha256 = "sha256-CPMcFzfP/l1g04sBLWj2pY76F94mNsr1RGom1sfY23I=";
+  };
+
+  # However we are only interested in building the specific crate located at `/ldproxy`
+  # cf: https://github.com/esp-rs/embuild/tree/v0.31.4/ldproxy
+  buildAndTestSubdir = "ldproxy";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    udev
+  ];
+
+  cargoSha256 = "sha256-dpdd4Yr92DrKeKrXx/OBkUQVo+bCNr+9VwXI7tvLAaU=";
```

Personally, I think it is best to put this attribute near the `src`, so that when a version bump comes, hashes are updated in the same diff, making it a bit smaller and easier to review. Also:

```suggestion
  cargoHash = "sha256-dpdd4Yr92DrKeKrXx/OBkUQVo+bCNr+9VwXI7tvLAaU=";
```

---

Potentially fixed in 0cd4466c2521dbdc3f6d6da63416985877e4c403